src/command.cpp

```diff
@@ -94,6 +94,8 @@ CommandProc CmdModifyOrder;
 CommandProc CmdSkipToOrder;
 CommandProc CmdDeleteOrder;
 CommandProc CmdInsertOrder;
+CommandProc CmdOrderAutomaticSeparation;
+
```

Comment:
Random extra newline :D Not sure if it was intended?

Reply:
Removed